volatility3/framework/plugins/layerwriter.py

```diff
@@ -95,15 +100,16 @@ def _generator(self):
             if not self.config["layers"]:
                 self.config["layers"] = []
                 for name in self.context.layers:
-                    if not self.context.layers[name].metadata.get("mapped", False):
+                    if "mapped" not in self.context.layers[name].metadata:
                         self.config["layers"] = [name]
 
             for name in self.config["layers"]:
                 # Check the layer exists and validate the output file
                 if name not in self.context.layers:
                     yield 0, (f"Layer Name {name} does not exist",)
                 else:
-                    output_name = self.config.get("output", ".".join([name, "raw"]))
+                    default_output_name = f"{name}.raw"
+                    output_name = self.config.get("output", default_output_name)
```

Comment:
You could use `output_name = self.config.get("output") or default_output_name`

Reply:
eehh .. yes, that will work but that's not very Pythonic... Python's get() has already a way to handle a default value and an experience Python developer will assume that behavior. Actually, whoever wrote line 106 already fall in this issue. IMO this is bad and prone to errors, as we can see here.

Reply:
It was probably me to be fair.  5;P  I think `or as default` is pythonic, but you're right it's not as clean as it could be and can lead to confusion.  Get and a default works better, but I'm not immediately seeing the issue with 106 that this new code resolves?